### PR TITLE
sistership PR for emonpi service file and install file harmonization

### PR DESCRIPTION
**Pull request by alexandrecuer**
simplification of install/emonpilcd.sh

needs this PR to be merged before : https://github.com/openenergymonitor/emonpi/pull/121

best

Alex

### CODE REVIEW

**Comment by borpin** [comment]
Looks good to me.

File: update/emonpi.sh [hunk 1]
```diff
@@ -17,3 +17,9 @@ avrdude -c arduino -p ATMEGA328P -P /dev/ttyAMA0 -b 115200 -U flash:w:$openenerg
 
 sudo service emonhub start
```
Comment (borpin) [inline update/emonpi.sh:18]: Should use systemctl now.

File: update/main.sh [hunk 2]
```diff
@@ -42,6 +42,9 @@ if [ "$emonSD_pi_env" = "1" ]; then
         # Stop emonPi LCD servcice
         echo "Stopping emonPiLCD service"
         sudo service emonPiLCD stop
```
Comment (borpin) [inline update/main.sh:44]: Use systemctl

File: update/emonpi.sh (hunk 1 above)
Comment (alexandrecuer) [reply update/emonpi.sh]: good point...done

File: update/main.sh (hunk 2 above)
Comment (alexandrecuer) [reply update/main.sh]: corrected also...thanks

**Comment by TrystanLea** [comment]
I've moved the emonpiLCD update to a new emonsd.sh script that's run at the end of the update process. The emonpi repository is also updated earlier here https://github.com/openenergymonitor/EmonScripts/blob/master/update/main.sh#L66

**Comment by TrystanLea** [comment]
Thanks for your help with this @alexandrecuer and @borpin

**Pull request merged**